### PR TITLE
MGDAPI-3552 update release doc

### PR DESCRIPTION
**Pull request by austincunningham**
## Overview

Jira: MGDAPI-3552
## Verification

Release Readme update

### CODE REVIEW

File: Update_CRO_integreatly-operator.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ add the latest version of CRO
 ## Release new version bundle of Cloud Resource Operator
 
 From your CRO branch, navigate to makefile and ensure that:
-- PREVIOUS_VERSION and VERSION match the desired versions.
+- PREVIOUS_VERSION and VERSION match the desired versions. You can confirm the PREVIOUS_VERSION by checking the last release CRO in rhmi_types
+version e.g. rhoam-release-v1.17.0 https://github.com/integr8ly/integreatly-operator/blob/rhoam-release-v1.17/apis/v1alpha1/rhmi_types.go#L96
```
Comment (MStokluska) [inline Update_CRO_integreatly-operator.md:9]: Do you think adding a note about "ensure that CRO has not been released in current sprint" makes sense?

Comment (austincunningham) [reply Update_CRO_integreatly-operator.md:9]: will add a note

**Comment by adam-cattermole** [comment]
When I followed the doc for the recent release I was going to make a ticket to update that link - didn't see this PR here. I realise there is no active ticket tracking this change but the changes look good.

/lgtm

**Pull request merged**